src/app/pipes/alm-valid-link-types.pipe.ts

```diff
@@ -0,0 +1,25 @@
+import { Pipe, PipeTransform } from '@angular/core';
```

Comment:
We need the pipe to filter the work item link types based on current work item type. For example if current work item is a bug, it should have only 'blocks' as link type list.